apps/balancer-tools/src/app/apr/round/(components)/PoolsCards.tsx

```diff
@@ -1,52 +1,62 @@
+import { networkFor } from "@bleu-balancer-tools/utils";
 import { ChevronRightIcon } from "@radix-ui/react-icons";
+import Link from "next/link";
+import { Suspense } from "react";
 
-import { DunePoolData } from "#/lib/dune";
+import { Gauge } from "#/lib/balancer/gauges";
+import { DuneGaugeData } from "#/lib/dune";
 import { fetcher } from "#/utils/fetcher";
 import { formatNumber } from "#/utils/formatNumber";
 
+import BALPrice from "../../(components)/BALPrice";
+
 function PoolCard({
   data: { symbol, pct_votes: pctVotes, votes },
 }: {
-  data: DunePoolData;
+  data: DuneGaugeData;
 }) {
+  const gauge = new Gauge(symbol);
+  const poolId = gauge.pool.id;
+
   return (
-    <div
-      key={symbol}
-      className="flex justify-between border border-gray-400 lg:border-gray-400 bg-blue3 rounded p-4 cursor-pointer"
-    >
-      <div className="">
-        <div className="flex justify-between">
-          <div className="text-white font-bold text-xl mb-2">{symbol}</div>
-        </div>
-        <div className="flex items-center">
-          <div className="text-sm">
-            <p className="text-white leading-none mb-1">
-              {pctVotes.toFixed(2)}% Voted
-            </p>
-            <p className="text-white text-xs">{formatNumber(votes)} Votes</p>
+    <Link href={`/apr/pool/${networkFor(gauge.network)}/${gauge.pool.id}`}>
+      <div className="flex justify-between border border-gray-400 lg:border-gray-400 bg-blue3 rounded p-4 cursor-pointer">
+        <div className="">
+          <div className="flex justify-between">
+            <div className="text-white font-bold text-lg mb-2">{poolId}</div>
+          </div>
+          <div className="flex items-center">
+            <div className="text-sm">
+              <p className="text-white leading-none mb-1">
+                {(pctVotes * 100).toFixed(2)}% Voted
+              </p>
+              <p className="text-white text-xs">{formatNumber(votes)} Votes</p>
+              <Suspense fallback={"Loading..."}>
+                <BALPrice />
```

Comment:
Isn't this going to display current value regardless which round is selected?

Reply:
ops, good catch!